corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -561,11 +561,10 @@ def data_source_properties(self):
 
     def _get_data_source_properties_from_case(self, case_properties):
         property_map = {
-            'closed': _('Case Closed'),
-            'user_id': _('User ID Last Updating Case'),
-            'owner_name': _('Case Owner'),
-            'mobile worker': _('Mobile Worker Last Updating Case'),
-            'case_id': _('Case ID')
+            'user_id': _('user_id_last_updating_case'),
+            'owner_name': _('owner_name'),
+            'mobile worker': _('mobile_worker_last_updating_case'),
```

Comment:
I'm not convinced that this is the right change. Why do these have underscores? I think they are labels, not properties. I would check with Omari and Kat what the canonical versions of this should be.

Reply:
This particular change was removed, but I think this comment also applies to this change: https://github.com/dimagi/commcare-hq/pull/28962/files#diff-14722173f99618e1204c85866b3d1960163f93bd8d2b3093a87c762cf86b1e9fR663 - @adil-uddin should that change still exist?